Person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize( age, name = "Unknown", parent_permission = true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    return true if @age >= 18
+    false
```

Comment:
Hi @Estete9,

Great work on the `Person` class! I noticed that the `of_age?` method is currently public, and as per the project requirements, it should be a private method. To address this, consider updating the method access level to private like this:

```ruby
private

def of_age?
  @age >= 18
end
```

This change adheres to the encapsulation principle, ensuring that the `of_age?` method is only accessible within the class. Keep up the good work, and feel free to reach out if you have any questions or need further clarification. Happy coding! 👍🚀